vboxlauncher@mockturtl/files/vboxlauncher@mockturtl/applet.js

```diff
@@ -13,7 +13,7 @@ const ICON = "virtualbox"
 const CMD_VBOX = "virtualbox"
 const CMD_VBOXMANAGE = "vboxmanage"
 
-const CMD_VBOX_VM = CMD_VBOX + " --startvm "
+const CMD_VBOX_VM = CMD_VBOXMANAGE + " startvm "
```

Comment:
This change makes `CMD_VBOX_VM` identical to `CMD_VBOX6_VM` (below).  Doesn't that break Virtualbox 5.x?
Notice vbox 6 support was introduced [here](https://github.com/linuxmint/cinnamon-spices-applets/commit/5751e8dd6fc92eb86c41da51b60f549d1117aff4#diff-41b8487c7ae21a4c12ec229ce7570857R17).

Reply:
I have VitrualBox 5.2.20 and everything is fine.

Reply:
OK.